modules/nw-egress-router-redirect-mode-ovn.adoc

```diff
@@ -16,11 +16,9 @@ After you add an egress router pod, the client pods that need to use the reserve
 
 .Procedure
 
-. Create a network attachment definition.
+. Create an egress router definition.
```

Comment:
Does it matter in what namespace this happens? Is EgressRouter cluster scoped?

Reply:
The namespace matters.  I've updated the YAML for the egress router def to show that it is possible to use `metadata.namespace`.  I used "Configuration for bridge" in the Multus area for the lingo.